recipes/seurat-scripts/conda_build_config.yaml

```diff
@@ -1,2 +1,2 @@
 r_base:
-  - 4.0
+  - 4.0.5
```

Comment:
Does it have to be that specific version? Would it break with any newer version, 4.0.6 ?

Reply:
This recipe has been a bit sensitive in the past, the format conversions parts have dependencies that easily break. The issue is probably not the r-base version, but the implications its version has on other packages (like the h5 related or loom).